t/stream/ssl.t

```diff
@@ -0,0 +1,1826 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua::Stream;use Cwd qw(cwd);
+
+#worker_connections(10140);
+#workers(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 6 + 1);
+
+our $CWD = cwd();
+
+no_long_string();
+#no_diff();
+
+env_to_nginx("PATH=" . $ENV{'PATH'});
+$ENV{TEST_NGINX_LUA_PACKAGE_PATH} = "$::CWD/lib/?.lua;;";
+$ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
+$ENV{TEST_NGINX_SERVER_SSL_PORT} ||= 4443;
```

Comment:
If we run the test suite concurrently, there is a chance that this suite conflicts with `t/ssl.t` (which uses the same env variable and defaults to the same port). Another approach would be to use a unix socket maybe?
```
listen unix:$ENV{TEST_NGINX_HTML_DIR}/nginx-ssl.sock ssl;
```

Reply:
That's a good idea, although does not really belongs to this PR. Maybe we can have a separate issue for it.

Reply:
I agree.

Reply:
@dndx It is a new issue introduced by this PR, so it's better to be addressed in this PR too? AFAIK, the current master does support running tests in parallel.